docs/reference/device-connector-types.rst

```diff
@@ -32,3 +32,52 @@ To specify the commands to run by the device in each test phase, set the ``testf
      - This device connector is used for Lenovo OEM devices running certain versions of OEM supported images that can use a recovery partition to recover not only the same image, but in some cases, other OEM image versions as well.
    * - ``hp_oemscript`` 
      - This device connector is used for HP OEM devices running certain versions of OEM supported images that can use a recovery partition to recover not only the same image, but in some cases, other OEM image versions as well.
+
+.. _muxpi:
+
+muxpi
+-----
+
+The ``muxpi`` device connector supports the following ``provision_data`` keys:
+
+.. list-table:: Supported ``provision_data`` keys for ``muxpi``
+   :header-rows: 1
+
+   * - Key
+     - Description
+   * - ``url``
+     - URL to a zstd-compressed disk image that is downloaded, decompressed and
```

Comment:
zstd handles multiple compression formats, but xz is typically used for images right now. I don't want to give the impression that it *must* be zstd compressed, so maybe we could change this a bit? Perhaps recommend xz but say that anything compatible with zstd is ok?